src/cascadia/TerminalApp/CommandPalette.h

```diff
@@ -33,7 +33,7 @@ namespace winrt::TerminalApp::implementation
         void SetTabActions(Windows::Foundation::Collections::IVector<Microsoft::Terminal::Settings::Model::Command> const& tabs, const bool clearList);
         void SetKeyBindings(Microsoft::Terminal::TerminalControl::IKeyBindings bindings);
 
-        void EnableCommandPaletteMode();
+        void EnableCommandPaletteMode(Microsoft::Terminal::Settings::Model::CommandPaletteLaunchMode const& launchMode);
```

Comment:
nit: no need for the `&` here since it's just as expensive to copy a reference as it is to copy a primitive :smile: